app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,76 @@
+class RecipesController < ApplicationController
+  before_action :authenticate_user!
+  before_action :set_recipe, only: %i[show edit update destroy]
+
+  # GET /recipes or /recipes.json
+  def index
+    @recipes = Recipe.all
+  end
+
+  # GET /recipes/1 or /recipes/1.json
+  def show; end

```

Comment:
- [ ] when I click on a recipe to navigate to the show page it gives me the following error please make sure to fix that 
<img width="1077" alt="Screenshot 2022-12-16 at 11 50 01" src="https://user-images.githubusercontent.com/61790587/208083156-511b671f-08c0-46fb-918a-b08c441f4e7f.png">

Reply:
Resolved